tests/python/ColorSpaceTest.py

```diff
@@ -1,31 +1,221 @@
-        lt = OCIO.LogTransform()
-        lt.setBase(10.0)
-        cs.setTransform(lt, OCIO.Constants.COLORSPACE_DIR_TO_REFERENCE)
-        ott = cs.getTransform(OCIO.Constants.COLORSPACE_DIR_TO_REFERENCE)
-        self.assertEquals(10.0, ott.getBase())
+    def setUp(self):
+        """
+        Set up self.cs as OCIO.ColorSpace and OCIO.LogTransform objects.
+        """
+
+        self.cs = OCIO.ColorSpace()
+        self.lt = OCIO.LogTransform()
+
+    def tearDown(self):
+        """
+        Clear the self.cs variable.
+        """
+
+        self.cs = None
+        self.lt = None
+
+    def test_editable(self):
+        """
+        Test the isEditable() and createEditableCopy() methods.
+        """
+
+        # Basic test
+        self.assertTrue(self.cs.isEditable())
+
+        cs_copy = self.cs.createEditableCopy()
+        self.assertTrue(isinstance(cs_copy, OCIO.ColorSpace))
+        self.assertTrue(cs_copy, self.cs)
+
+    def test_name(self):
+        """
+        Test the setName() and getName() methods.
+        """
+
+        # Basic test
+        self.cs.setName("mynewcolspace")
+        self.assertEqual("mynewcolspace", self.cs.getName())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(10)
+            self.cs.setName(name)
+            self.assertEqual(name, self.cs.getName())
+
+    def test_family(self):
+        """
+        Test the setFamily() and getFamily() methods.
+        """
+
+        # Basic test
+        self.cs.setFamily("fam1")
+        self.assertEqual("fam1", self.cs.getFamily())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(10)
+            self.cs.setFamily(name)
+            self.assertEqual(name, self.cs.getFamily())
+
+    def test_equality(self):
+        """
+        Test the setEqualityGroup() and getEqualityGroup() methods.
+        """
+
+        # Basic test
+        self.cs.setEqualityGroup("match1")
+        self.assertEqual("match1", self.cs.getEqualityGroup())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(10)
+            self.cs.setEqualityGroup(name)
+            self.assertEqual(name, self.cs.getEqualityGroup())
+
+    def test_description(self):
+        """
+        Test the setDescription() and getDescription() methods.
+        """
+
+        # Basic test
+        self.cs.setDescription("this is a test")
+        self.assertEqual("this is a test", self.cs.getDescription())
+
+        # Random string tests
+        for i in range(1, 10):
+            name = generate_name(20)
+            self.cs.setDescription(name)
+            self.assertEqual(name, self.cs.getDescription())
+
+    def test_bitdepth(self):
+        """
+        Test the setBitDepth() and getBitDepth() methods.
+        """
+
+        # Known constants tests
+        bit_depths = [OCIO.Constants.BIT_DEPTH_UNKNOWN,
```

Comment:
In the new pybind bindings you *should* be able to iterate on the enum instead of listing all the possible values here, as an example of somrhting that will change/improve.